README.md

```diff
@@ -8,7 +8,7 @@ Example:
 GET
 ```java
 HttpRequest request = httpRequest(new TargetUrl("abv.bg")).build();
-ChitaHttp chitaClient = new ChitaHttpClient();// in most cases chitaClient should be injected
+ChitaHttp chitaClient = new HttpClient();// in most cases chitaClient should be injected
```

Comment:
Ok, so, as I understand, you want us to write test using test servers and to remove the interface ChitaHttp.